torchaudio/datasets/voxceleb1.py

```diff
@@ -135,6 +135,17 @@ class VoxCeleb1Identification(VoxCeleb1):
             (Default: ``"https://www.robots.ox.ac.uk/~vgg/data/voxceleb/meta/iden_split.txt"``)
         download (bool, optional):
             Whether to download the dataset if it is not found at root path. (Default: ``False``).
+
+    Note:
+        The file structure of `VoxCeleb1Identification` dataset is as follows:
+            root/
+
```

Comment:
nit: would it look better if the blank lines are removed? do you have the rendered output for this

Reply:
i see, what if you remove one of the `_`, to make it a bit narrower?